src/app/games/diaballik/diaballik.component.html

```diff
@@ -50,10 +56,89 @@
               (click)="onClick(coord.x, coord.y)"
               class="indicator"/>
     </g>
+    <g *ngIf="isInteractive"
+       [attr.transform]="getTranslationXY(0, HEIGHT + 0.3) + ' scale(0.5)'">
+        <rect class="base"
+              [attr.x]="0"
+              [attr.y]="0"
+              [attr.width]="SPACE_SIZE"
+              [attr.height]="SPACE_SIZE"
+        />
+        <circle class="base"
```

Comment:
aussi, devrais-ce être visible en fin de jeu ?

Reply:
bonne question, je dirais que si non, ça signifie que isInteractive devrait être false à la fin du jeu ! Et ce n'est pas le cas, donc à ticketter pour généraliser sur tous les jeux ? (et voir si c'est vraiment ça qu'on souhaite)

Reply:
yep tu peux ticketter que si c'est à personne de jouer c'est interactif pour personne.
bien évidemment ?

Reply:
[W] [easy] GameComponent: isInteractive - when the game is ended, it should be set to false